composer.json

```diff
@@ -15,7 +15,7 @@
         "henrikbjorn/stampie"      : "~0.11"
     },
     "require-dev": {
-        "stof/stampie-extra": "~0.2@dev",
+        "stof/stampie-extra": "^1.0@beta",
```

Comment:
This should probably be in #23.

Reply:
You are correct. Let’s merge #23 first and I’ll rebase this later